.github/ISSUE_TEMPLATE/00-bug.md

```diff
@@ -50,5 +50,9 @@ e.g. [`test/path/to/foo.chpl`](
 
 - Output of `chpl --version`:
 - Output of `$CHPL_HOME/util/printchplenv --anonymize`:
+  <!--
+  When `$CHPL_HOME` is not set, you can use `chpl --print-chpl-settings`
+  instead, though be aware that this does not anonymize the output (yet)
```

Comment:
Are we actually planning on having `--print-chpl-settings` anonymize the output?  If not, then I recommend taking out the "(yet)".

Reply:
I think that's open for debate (was in the process of writing #25018 as you were reviewing :) )

Reply:
A temporary workaround for this is `$(chpl --print-chpl-home)/util/printchplenv --anonymize`

Reply:
Do you think that's better or worse than providing the second option?  To me it feels like it's starting to get a little wordy.  And it looks more like an inscrutable bash command, which might make it more intimidating for users not as familiar with the command prompt.  But I could also see a reader misinterpreting what I've currently written as meaning they should write `chpl --print-chpl-settings/util/printchplenv --anonymize` or something along those lines.  So I'm personally torn

Reply:
I agree that `$(chpl --print-chpl-home)/util/printchplenv --anonymize` is not pretty, and it really just feels like we should quickly implement #25018, especially because `--print-chpl-settings` is just calling `printchplenv`.

Reply:
I think I would just leave it as "When `$CHPL_HOME` is not set, you can use `chpl --print-chpl-settings` instead, though be aware that this does not anonymize the output" to keep it simple